### PR TITLE
Updated AutomationAccount with `existing`

### PR DESCRIPTION
**Pull request by MrMCake**
# Change

- Update docs & usage of `existing`
- Minor improvements

Pipeline reference
[Automation: Automationaccounts](https://github.com/Azure/ResourceModules/actions/workflows/ms.automation.automationaccounts.yml)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

### CODE REVIEW

**Review by eriqua — DISMISSED** [review]
I'd suggest to put this in draft as many updates are already covered by @MariusStorhaug PR

File: arm/Microsoft.Automation/automationAccounts/schedules/deploy.bicep [hunk 1]
```diff
@@ -57,13 +57,13 @@ resource schedule 'Microsoft.Automation/automationAccounts/schedules@2020-01-13-
...
-    expiryTime: (empty(expiryTime) ? null : expiryTime)
-    frequency: (empty(frequency) ? 'OneTime' : frequency)
-    interval: ((0 == interval) ? null : interval)
-    startTime: (empty(startTime) ? dateTimeAdd(baseTime, 'PT10M') : startTime)
-    timeZone: (empty(timeZone) ? null : timeZone)
+    advancedSchedule: !empty(advancedSchedule) ? advancedSchedule : null
+    description: !empty(scheduleDescription) ? scheduleDescription : null
+    expiryTime: !empty(expiryTime) ? expiryTime : null
+    frequency: !empty(frequency) ? frequency : 'OneTime'
+    interval: (0 != interval) ? interval : null
```
Comment (MariusStorhaug) [inline arm/Microsoft.Automation/automationAccounts/schedules/deploy.bicep:64]: Reads better, and its lot having the same 'got-ya' as PowerShell.
```suggestion
    interval: (interval != 0) ? interval : null
```

**Review by MariusStorhaug — CHANGES_REQUESTED** [review]

File: arm/Microsoft.Automation/automationAccounts/schedules/deploy.bicep (hunk 1 above)
Comment (MrMCake) [reply arm/Microsoft.Automation/automationAccounts/schedules/deploy.bicep]: done

**Review by MariusStorhaug — APPROVED** [review]
Fixes are done. Approved.

**Pull request merged**